packages/next/src/build/swc/index.ts

```diff
@@ -1059,122 +1078,165 @@ function bindingToApi(
-              )
-            },
-            transformStyleAttr: function (_options: any) {
-              throw new Error(
-                '`css.lightning.transformStyleAttr` is not supported by the wasm bindings.'
-              )
-            },
-          },
-        },
-        isWasm: true,
-        transform(src: string, options: any) {
-          // TODO: we can remove fallback to sync interface once new stable version of next-swc gets published (current v12.2)
-          return bindings?.transform
-            ? bindings.transform(src.toString(), options)
-            : Promise.resolve(bindings.transformSync(src.toString(), options))
-        },
-        transformSync(src: string, options: any) {
-          return bindings.transformSync(src.toString(), options)
-        },
-        minify(src: string, options: any) {
-          return bindings?.minify
-            ? bindings.minify(src.toString(), options)
-            : Promise.resolve(bindings.minifySync(src.toString(), options))
-        },
-        minifySync(src: string, options: any) {
-          return bindings.minifySync(src.toString(), options)
-        },
-        parse(src: string, options: any) {
-          return bindings?.parse
-            ? bindings.parse(src.toString(), options)
-            : Promise.resolve(bindings.parseSync(src.toString(), options))
-        },
-        getTargetTriple() {
-          return undefined
-        },
-        turbo: {
-          createProject: function (
-            _options: ProjectOptions,
-            _turboEngineOptions?: TurboEngineOptions | undefined
-          ): Promise<Project> {
-            throw new Error(
-              '`turbo.createProject` is not supported by the wasm bindings.'
-            )
-          },
-          startTurbopackTraceServer: function (_traceFilePath: string): void {
-            throw new Error(
-              '`turbo.startTurbopackTraceServer` is not supported by the wasm bindings.'
-            )
-          },
-        },
-        mdx: {
-          compile(src: string, options: any) {
-            return bindings.mdxCompile(src, getMdxOptions(options))
-          },
-          compileSync(src: string, options: any) {
-            return bindings.mdxCompileSync(src, getMdxOptions(options))
-          },
-        },
-        reactCompiler: {
-          isReactCompilerRequired(_filename: string) {
-            return Promise.resolve(true)
-          },
-        },
-      }
-      return wasmBindings
-    } catch (e: any) {
-      // Only log attempts for loading wasm when loading as fallback
-      if (importPath) {
-        if (e?.code === 'ERR_MODULE_NOT_FOUND') {
-          attempts.push(`Attempted to load ${pkg}, but it was not installed`)
+  // Used by `run-tests` to force use of a locally-built wasm binary. This environment variable is
+  // unstable and subject to change.
+  const testWasmDir = process.env.NEXT_TEST_WASM_DIR
+
+  if (testWasmDir != null) {
```

Comment:
can this actually be null? process.env values are meant to be `string | undefined` so plain truthiness seems to be enough (unless we're doing something weird with this env var somewhere?)
```suggestion
  if (testWasmDir) {
```

Reply:
I was doing this because I wanted to explicitly check for `null` or `undefined`. I know this can never be `null`, but I just default to `!= null` instead of `!== undefined`.

A simple truthiness/falsiness check is fine too.